request/middleware.py

```diff
@@ -3,8 +3,12 @@
 from request.models import Request
 from request.router import patterns
 
-
-class RequestMiddleware(object):
+try:
+	from django.utils.deprecation import MiddlewareMixin
+except ImportError:
+	MiddlewareMixin = object
```

Comment:
Can you add a comment? For example `to keep Django <= 1.9 - MIDDLEWARE_CLASSES and  Django >= 1.10 - MIDDLEWARE compatibility`